src/util/Env.ts

```diff
@@ -0,0 +1,10 @@
+const DEVMODE: boolean = (process.env.REACT_APP_ENV === "development");
+const IS_DEV = DEVMODE;
+const IS_PROD = !DEVMODE;
+
+console.log(`Devmode? ${IS_DEV}`);
```

Comment:
I would remove this before putting it into production

Reply:
oh shoot, missed this. Thanks for the catch.